src/actions/public/call.ts

```diff
@@ -169,7 +200,7 @@ export async function call<TChain extends Chain | undefined>(
       value,
     } as TransactionRequest) as TransactionRequest
 
-    if (batch && shouldPerformMulticall({ request })) {
+    if (!rawStateOverride && batch && shouldPerformMulticall({ request })) {
```

Comment:
Might read better after the `batch` condition. 

"If we have **batching enabled**, and blah blah blah"

```suggestion
    if (batch && shouldPerformMulticall({ request }) && !rawStateOverride) {
```